pkg/client/logging/initcontext_test.go

```diff
@@ -56,15 +55,12 @@ func TestInitContext(t *testing.T) {
 		// os.Stdout/os.Stdin; so they need to be backed up and restored.
 		saveStdout := os.Stdout
 		saveStderr := os.Stderr
-		stdoutFd, err := unix.Dup(1)
-		require.NoError(t, err)
-		stderrFd, err := unix.Dup(2)
+		restoreStd, err := dupStd()
 		require.NoError(t, err)
 		t.Cleanup(func() {
 			os.Stdout = saveStdout
 			os.Stderr = saveStderr
-			_ = unix.Dup2(stdoutFd, 1)
-			_ = unix.Dup2(stderrFd, 2)
+			restoreStd()
 		})
```

Comment:
Backported from @josecv's PR (well, plus my feedback on that PR) to get `make lint` to work.